notebooks/community/neo4j/graph_paysim.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        df = pd.DataFrame(result)
</pre><p>Across this notebook, the dataframe assignment and display occurs both within and outside <code>with</code>  statements. Consider following the pattern of the 2nd query, where it is outside.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Done